src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -1087,6 +1091,10 @@ internal override Cci.IMethodReference Translate(MethodSymbol symbol, Diagnostic
             {
                 methodSymbol = AnonymousTypeManager.TranslateAnonymousTypeMethodSymbol(methodSymbol);
             }
+            else if (container.IsTupleType)
+            {
+                container = ((TupleTypeSymbol)container).UnderlyingTupleType;
```

Comment:
I thought about that, but decided not to touch that for now. We currently rewrite fields during lowering, but I think we might have to stop doing that. Debugging and EnC scenarios might require that.